README.md

```diff
@@ -1,92 +1,88 @@
-<p align="center">
-  <img alt="ECMA-V banner" src="https://yowainwright.imgix.net/gh/ecma-v.svg" width="150" />
-</p>
-
-<h1 align="center">ECMA-V is now called ES-Check. Chekout out <a href="https://github.com/dollarshaveclub/es-check">ES-Check</a>!<h1>
+<h1 align="center">ES Check ✔️<h1>
 
 ----
 
-<h5 align="center">Checks the version of ECMAScript used in JavaScript with shell commands  🏆</h5>
+<h5 align="center">Checks the JavaScript Files version of ECMAScript (ES) with shell commands  🏆</h5>
 
 ----
 
 <p align="center">
   <a href="#api">API</a>&nbsp;&nbsp;
   <a href="#usage">Usage</a>&nbsp;&nbsp;
-  <a href="#why">Why ECMA V?</a>&nbsp;&nbsp;
+  <a href="#why">Why es check?</a>&nbsp;&nbsp;
   <a href="#contributing">Contributing</a>&nbsp;&nbsp;
   <a href="/issues">Issues</a>  
 </p>
 
 ----
 
-**ECMA V**, short for `ECMA Version`, checks JavaScript files against a specified version of ECMAScript with a shell command. If a specified file's ECMAScript version doesn't match the ECMAScript version argument passed in the Ecma V shell script, Ecma V will throw an error.  In example, by running the command `ecma-v check ecma6 ./dist/*.js` will check `/dist/*.js` files to see if they're ECMAScript 5 and throw errors for any files are that are not. This is important in a [modular and bundled](https://www.sitepoint.com/javascript-modules-bundling-transpiling/) world. 
+**ES Check** checks JavaScript files against a specified version of ECMAScript with a shell command. If a specified file's ECMAScript version doesn't match the ECMAScript version argument passed in the es check shell script, es check will throw an error.  In example, by running the command `es-check check ecma5 ./dist/*.js` will check `/dist/*.js` files to see if they're ECMAScript 5 and throw errors for any files are that are not. This is important in a [modular and bundled](https://www.sitepoint.com/javascript-modules-bundling-transpiling/) world. 
```

Comment:
`In example` -> `For example`

remove `by running the command ` (not needed)

Reply:
🙏